internal/ls/utilities.go

```diff
@@ -41,6 +86,58 @@ func tryGetImportFromModuleSpecifier(node *ast.StringLiteralLike) *ast.Node {
 	return nil
 }
 
+func isModuleSpecifierLike(node *ast.Node) bool {
+	// return node.Kind == ast.KindStringLiteral || node.Kind == ast.KindNoSubstitutionTemplateLiteral ||
```

Comment:
If this was just a refactoring replacing the explicit return with `ast.IsStringLiteralLike`, I'd just get rid of the comment.